content/W2015/W15-event-research-showcase.md

```diff
@@ -1,7 +1,7 @@
 Title: Research Showcase and Mixer
 Date: 2015-02-04 17:30
 Category: Events
-Tags: talks, academia, social
+Tags: talks, academic, social
```

Comment:
Personally, I prefer "academic" and it seems to be what people have actually been using so I vote we go for that.